appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap

```diff
@@ -316,7 +316,7 @@ exports[`appellant-case GET /appellant-case with unchecked documents should rend
         <div class="govuk-summary-card__content">
             <dl class="govuk-summary-list">
                 <div class="govuk-summary-list__row"><dt class="govuk-summary-list__key"> Appellant name</dt>
-                    <dd class="govuk-summary-list__value">Fiona Burgess</dd>
+                    <dd class="govuk-summary-list__value"></dd>
```

Comment:
This looks like the test data just needs updating to reflect the change (in `appeals/web/testing/app/fixtures/referencedata.js`)

Reply:
Well spotted! Added another commit...